app/main.py

```diff
@@ -1,8 +1,32 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person_data in people:
+        person_list = []
+
+        for person_data in people:
+            name = person_data["name"]
+            age = person_data["age"]
+            person = Person(name, age)
```

Comment:
This code can be written in one line: person = Person(name=person_data["name"], age=person_data["age"])